lib/console.js

```diff
@@ -90,5 +96,44 @@ Console.prototype.assert = function(expression) {
 };
 
 
+Console.prototype.group = function() {};
+
+
+Console.prototype.groupCollapsed = function() {};
+
+
+Console.prototype.groupEnd = function() {};
+
+
+Console.prototype.clear = function() {};
+
+
+Console.prototype.count = function() {};
+
+
+Console.prototype.dirxml = function() {};
+
+
+Console.prototype.isIndependentlyComposed = function() {};
+
+
+Console.prototype.table = function() {};
+
+
+Console.prototype.profile = function() {};
+
+
+Console.prototype.profileEnd = function() {};
+
+
+Console.prototype.timeline = function() {};
+
+
+Console.prototype.timelineEnd = function() {};
+
+
+Console.prototype.timeStamp = function() {};
```

Comment:
Perhaps we can define a `noop` function once in the file, and then assign them all to it.